### ISSUE TITLE
Thorough-walkthrough user guide aren't arranged in order, wrt to navigation bar

### ISSUE THREAD

**Issue by janetkuo** [kubernetes/kubernetes.github.io#192]
In [user-guide/#thorough-walkthrough](http://kubernetes.io/docs/user-guide/#thorough-walkthrough]), we recommended the users to read a number of docs in order. 

However, each referenced doc is in different section, with different order on left navigation bar, and it feels hard to follow. 

For example, the first doc, [Quick start: launch and expose an application](http://kubernetes.io/docs/user-guide/quick-start/) is under Application Management and Deployment > Launching, Exposing, and Killing Applications; and the second one, [Configuring and launching containers: configuring common container parameters](http://kubernetes.io/docs/user-guide/configuring-containers/), is under Configuring Kubernetes > Configuring Containers. 

It'd be great if tutorials / getting started guides are grouped in order like this https://cloud.google.com/python/getting-started/hello-world

@johndmulhausen @bgrant0607 @pwittrock @kubernetes/docs

**Comment by jaredbhatti** [comment]
Yep, in progress with our migration.
